client/src/components/App.js

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react';
+import logo from '../assets/logo.svg';
+
+import Display from './Display'
+import Dashboard from './Dashboard'
+
+
+class App extends Component {
+  state = {
+    atBat: 1,
+    balls: 0,
+    strikes: 0,
+    strikeout: false,
+    walk: false,
+    numStrikeouts: 0,
+    numWalks: 0,
+    inning: 1,
+    hits: 0,
+    foulBalls: 0,
+  }
+
+  hitBall = () => {
+    if (this.state.balls === 3) {
+      return this.setState({
+        ...this.state,
+        balls: 0,
```

Comment:
careful, you also want to reset the number of `strikes` to 0 as well -- and within your hitStrike function, once three strikes have been met, you also want to reset the whole count and not just strikes